template_content/pyproject.toml.jinja

```diff
@@ -7,25 +7,25 @@ readme = "README.md"
 
 [tool.poetry.dependencies]
 python = "^3.12"
-algokit-utils = "^2.4.0"
+algokit-utils = "^3.0.0b5"
 python-dotenv = "^1.0.0"
 algorand-python = "^2.0.0"
 algorand-python-testing = "^0.4.0"
 
 [tool.poetry.group.dev.dependencies]
 {% if deployment_language == 'python' -%}
-algokit-client-generator = "^1.1.3"
+algokit-client-generator = "^2.0.0b4"
```

Comment:
I'm assuming we'll hold off on merging this PR until we've shipped utils + generator.

Reply:
@neilcampbell yes